openapi/installinator-artifactd.json

```diff
@@ -53,57 +53,6 @@
         }
       }
     },
-    "/artifacts/by-id/{kind}/{name}/{version}": {
```

Comment:
Note to self / future readers: this _would_ be a backwards incompatible change (installinator talks to wicketd), except no client was actually using this endpoint.